jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -591,6 +591,16 @@ properties:
             type: integer
             fits_keyword: EXSEGTOT
             blend_table: True
+          integration_start:
+            title: Integration number at start of segment
+            type: integer
+            fits_keyword: INTSTART
+            blend_table: True
+          integration_end:
+            title: Integration number at end of segment
+            type: integer
+            fits_keyword: INTEND
+            blend_table: True
```

Comment:
Whoops, looks like we've both made changes to add these keywords (see #1989). They're very close to being identical changes, but I've used titles in #1989 that match the specs for what's being added to the dictionary (https://jira.stsci.edu/browse/JWSTKD-216).

Reply:
Yes, your changes should override mine.